ral-facilities/gridftp_acl_plugin
# Feature/new solution #9

Set up an initial solution that prints hello world to the console and the log.

Steps to test:
- Review the dev environment instructions on the wiki.
- Use the wiki instructions to build and test the code.
- Check that a log message appears in `/usr/tmp/gridftp_acl_plugin.log`

connect to #9 
closes #9
